recipes/mdns/all/conanfile.py

```diff
@@ -0,0 +1,41 @@
+import os
+from conans import CMake, ConanFile, tools
+
+
+class MdnsConan(ConanFile):
+    name = "mdns"
+    license = "Public Domain or MIT"
```

Comment:
```suggestion
    license = "Unlicense"
```

https://github.com/mjansson/mdns/blob/master/LICENSE